client/jetpack-cloud/sections/overview/primary/overview-products/jetpack-products.ts

```diff
@@ -0,0 +1,49 @@
+import { translate } from 'i18n-calypso';
+
+export interface ProductData {
+	description: string;
+	url: string;
+	urlText?: string;
+	product_slug?: string;
+	data?: any;
+	name?: string;
+}
+export interface JetpackProducts {
+	[ key: string ]: ProductData;
+}
+
+// It will show in this order: left to right, top to down, 2x4 grid.
```

Comment:
Ideally this would be in the API response, but it's not, so this works for now...